src/Console/Command/RunCommand.php

```diff
@@ -66,9 +73,12 @@ public function execute(InputInterface $input, OutputInterface $output)
         $files = $this->getRegisteredFiles();
         $testSuites = $this->grumPHP->getTestSuites();
 
+        $tasks = $this->parseCommaSeparatedOption($input->getOption("tasks"));
```

Comment:
```suggestion
        $tasks = array_map('trim', explode(',', $input->getOption('tasks', '')));
```

The de-duplicating logic is a bit confusing and is not required since the filter method on the taskcollection is smart enough. I would prefer to drop it for the single line suggested above. (optionally you could add array_filter to drop the empty items, but that is also covered by the filter method)

Reply:
This will retain empty input values, i.e. if `--tasks` is empty or omitted, `$tasks` will be an array containing one empty element. So we would either put it in an if OR add and `array_filter` --- both makes the result less readable, imho.

I'm fine with removing deduplication, though.

Thoughts?

fyi - failing tests after change:
```
/codebase/grumphp/vendor/phpunit/phpunit/phpunit --configuration /codebase/grumphp/phpunit.xml.dist GrumPHPTest\\Console\\Command\\RunCommandTest /codebase/grumphp/test/Console/Command/RunCommandTest.php

[...]
4) GrumPHPTest\Console\Command\RunCommandTest::parses_comma_separated_options with data set "null" (null, array())
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
+    0 => ''
 )

/codebase/grumphp/test/Console/Command/RunCommandTest.php:36

5) GrumPHPTest\Console\Command\RunCommandTest::parses_comma_separated_options with data set "empty" ('', array())
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
+    0 => ''
 )

/codebase/grumphp/test/Console/Command/RunCommandTest.php:36

6) GrumPHPTest\Console\Command\RunCommandTest::parses_comma_separated_options with data set "empty after trim" (' ', array())
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
+    0 => ''
 )
```